docs/modules/icon.md

```diff
@@ -48,3 +48,12 @@ Will return an empty string when no icon in the given size is available (eg. whe
 - Default: '$icon'
 
 Name of property for accessible icons.
+
+**purpose**
+- Default: null
+
+Array of icon purpose.
```

Comment:
Can you please improve docs by putting more description or links?

Reply:
I added some description, please review.